From 8fd54b4fba1f81c788064cc9cbb2361217b6b19a Mon Sep 17 00:00:00 2001
From: Wenyou Yang <wenyou.yang@atmel.com>
Date: Thu, 4 Dec 2014 11:02:22 +0800
Subject: [PATCH] Config: NOR flash: indicate support to boot from the NOR
 flash is depended on the ROM Code version

Booting from the NOR flash is supported form ROM Code v2.1.

Signed-off-by: Wenyou Yang <wenyou.yang@atmel.com>
---
 driver/Config.in.memory | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/driver/Config.in.memory b/driver/Config.in.memory
index 56f0302..130ca76 100644
--- a/driver/Config.in.memory
+++ b/driver/Config.in.memory
@@ -106,8 +106,12 @@ config	CONFIG_DATAFLASH
 	depends on ALLOW_DATAFLASH
 
 config	CONFIG_FLASH
-	bool "Parallel NOR flash"
+	bool "Parallel NOR flash(supported from ROM Code v2.1)"
 	depends on ALLOW_FLASH
+	help
+	  Before you select it, please check the ROM Code if it supports
+	  to boot from the NOR flash.  Beacause booting from the external
+	  NOR memory is supported in ROM Code v2.1 for SAMA5D3x.
 
 config	CONFIG_NANDFLASH
 	bool "NAND flash"
-- 
1.9.1

